docs/react-testing-library/setup.md

```diff
@@ -144,6 +144,35 @@ module.exports = {
 
 </details>
 
+<details>
+<summary>Modify queries globally with custom render</summary>
+
+You can override and append queries via the render function by passing a [`queries`](api.md#render-options) option.
+
+If you want to add custom queries globally, you can do this by defining a custom render method:
+
+```js
+// test-utils.js
+import { render, queries, queryHelpers } from '@testing-library/react'
+
+const customQueries = {
+  getByDataCy: queryHelpers.queryByAttribute.bind(
```

Comment:
It would be nice to show how to use the `buildQueries` function to create a full set of custom queries that conforms to the `get`/`query`/`find` convention

https://github.com/testing-library/dom-testing-library/blob/master/src/query-helpers.js#L103

Reply:
Great suggestion - I'll look into it.

Reply:
@alexkrolick do you reckon using `buildQueries` in this example specifically would be useful, or shall I get an example added to the 'Helpers' docs for `dom-testing-library`?

Reply:
The function should be documented in the API section and it can point to a usage example here

Reply:
@alexkrolick added example usage, along with a brief description in the API section

Reply:
@alexkrolick any further comments following my updates?